### PR TITLE
Bulk construct correlations plate 4

### PR DESCRIPTION
**Pull request by MattsonCam**
This analysis finds the aggregate correlation pdfs across wells for both the same and different siRNA constructs with some refactoring. The `no_treatment` type is also compared in this analysis. The `no_treatment` type will only have one pdf, since no other constructs have a zero concentration, otherwise known as `no_treatment`. The filtering operations to compute the correlation distributions are complex, so feel free to ask any questions. Also, further analysis is warranted, and is to be continued.

### CODE REVIEW

**Review by gwaybio — APPROVED** [review]
LGTM! I have a couple of questions, but they need not delay merging.

1. Do you know why there was such a dramatic shift down in correlations? What specifically changed between your median aggregation and Jenna's?
2. It would be helpful to include a README that describes which comparisons you're making. For the scramble control and no_treatment, are you comparing them to all other wells or all other wells based on dose? Which wells are you excluding?

**Comment by MattsonCam** [comment]
Hey @gwaybio, the analysis should be updated. Please let me know if you have any other feedback.

File: 0.data_analysis/construct_phenotypic_expression_plate_4_fs_data/median_correlation_relationships/README.md [hunk 1]
```diff
@@ -1,8 +1,17 @@
...
 - Different siRNA constructs within each construct concentration
 
-Correlations were then plotted and compared by estimating the pdfs using kde for each construct concentration for the above groups.
+The `no_treatment` wells were only compared with other `no_treatment` wells for this analysis.
+
+### Visualzing Comparisons
+Correlations were then plotted as boxplots for each concentration in the following groups:
+
+- (Construct 1) and (Construct 2)
+- (Construct 1) and (Constrcut 1)
```
Comment (gwaybio) [inline 0.data_analysis/construct_phenotypic_expression_plate_4_fs_data/median_correlation_relationships/README.md:14]: How are these different? Shouldn't they have the exact same correlations? I am seeing the boxplots below are different, and I do not understand why

**Review by gwaybio — APPROVED** [review]
Nice, easy to review PR! well done! I had one question, which may result in additional code changes.

File: 0.data_analysis/construct_phenotypic_expression_plate_4_fs_data/median_correlation_relationships/README.md (hunk 1 above)
Comment (MattsonCam) [reply 0.data_analysis/construct_phenotypic_expression_plate_4_fs_data/median_correlation_relationships/README.md:14]: Construct 1 and Construct 2 should have different off target effects, so I would expect the comparisons between respective features to vary more compared to the same constructs, leading to some difference between the correlation distributions, although, this also depends on how much the well conditions vary.

Comment (gwaybio) [reply 0.data_analysis/construct_phenotypic_expression_plate_4_fs_data/median_correlation_relationships/README.md:14]: Gotcha! I think I initially read the comparisons wrong.

I thought that there were two different boxplots for the following groups:

1. Construct 1 vs. Construct 2
2. Construct 2 vs. Construct 1

Theoretically, these should have the exact same values, if I'm understanding your procedure.

I think we're all clear now though! I will go ahead and merge this PR, and then update the figure in #11 

Thanks!

**Pull request merged**